lightning_talk.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'talk'
+
+class LightningTalk < Talk
```

Comment:
I feel this inheritance here is overkill as there are no extra methods or props added, can't 2 objects of session do the same?
we can have an enum in talk with 2 types to do the same.

Reply:
Yeah, it was a bit. But I just played around with inheritance a bit in this assignment as I didn't know where to apply all my knowledge.